pack/tcu/bbt_encounter.json

```diff
@@ -0,0 +1,404 @@
+		"back_name": "The End",
+		"back_text": "Check Campaign Log.\n- If at least two of the following are true, you may proceed to <b>(→R2)</b>: <i>You accepted your fate</i>, <i>Mesmirizing Flute</i> is listed under \"Mementos Discovered,\" and <i>Ritual Components</i> is listed under \"Mementos Discovered.\"\n- If at least two of the following are true, you may proceed to <b>(→R3)</b>: <i>You rejected your fate</i>, <i>Scrap of Torn Shadow</i> is listed under \"Mementos Discovered,\" and <i>Wisp of Spectral Mist</i> is listed under \"Mementos Discovered.\"\n- Otherwise, <b>(→R4)</b>.",
+		"clues": 0,
+		"code": "05331",
+		"double_sided": true,
+		"encounter_code": "before_the_black_throne",
+		"faction_code": "mythos",
+		"flavor": "You are close now. You can feel the churning and twisting of aeons in the emptiness around you. Here, in the nucleus of all things—the very center of chaos itself—your objective, and your ultimate fate, lie at hand.",
+		"illustrator": "Reiko Murakami",
+		"name": "What Must Be Done",
+		"pack_code": "bbt",
+		"position": 331,
+		"quantity": 1,
+		"stage": 3,
+		"text": "[action] Spend X clues: Draw the top X cards of the Cosmos. Choose one to put into play using its \"<b>Cosmos</b> -\" instructions, and move to it. Shuffle the rest back into the Cosmos.\n<b>Objective</b> - If the lead investigator is at The Black Throne and there are no clues on it, advance.",
+		"type_code": "act"
+	},
+	{
+		"back_flavor": "...that eldritch scaly monster bore its helpless rider; hurtling and shooting, cleaving the uttermost rim and spanning the outermost abysses; leaving behind the stars and the realms of matter, and darting meteor-like through stark formlessness toward those inconceivable, unlighted chambers beyond Time...\n– H. P. Lovecraft, The Dream-Quest of Unknown Kadath",
+		"clues": 3,
+		"clues_fixed": true,
+		"code": "05332",
+		"double_sided": true,
+		"encounter_code": "before_the_black_throne",
+		"faction_code": "mythos",
+		"illustrator": "Juan Carlos Barquet",
+		"name": "Cosmic Ingress",
+		"pack_code": "bbt",
+		"position": 332,
+		"quantity": 1,
+		"shroud": 2,
+		"text": "<b>Forced</b> - At the end of the round: Add clues to Cosmic Ingress until it has 3 clues on it.\nEach [[Void]] location gains: \"[action]: Shuffle this location into the Cosmos, moving each investigator and enemy that was at this location to Cosmic Ingress.\"",
+		"traits": "Otherworld.",
+		"type_code": "location"
+	},
+	{
+		"back_illustrator": "Uwe Jarling",
+		"back_name": "Comsos",
+		"back_flavor": "The stars danced mockingly, almost shifting now and then to form pale signs of doom that one might wonder one had not seen and feared before; and ever the winds of aether howled of vague blackness and loneliness beyond the cosmos.\n– H. P. Lovecraft, The Dream-Quest of Unknown Kadath",
```

Comment:
Do we like to do underline tags for story names? Happy to leave it if you don't fancy adding it across all the cosmos locations!